### ISSUE TITLE
iOS Feature Request

### ISSUE THREAD

**Issue by shad0whawk** [strongbox-password-safe/Strongbox#225]
It would be extremely useful if I could limit my search to a specific point in the folder hierarchy and all of its subfolders without it searching the parent folders. My use case was that I was trying to find a particular data attribute within a folder and it’s subfolders about three levels down. It exists in other folders but should not exist in this folder. I navigated to that specific folder and then pulled down to display the search bar. I was hoping it would search from that point on down but it didn’t. Is this a possibility?

**Comment by mmcguill** [comment]
Yes very possible, perhaps by an extra scope button alongside title, username, password etc
